scripts/base/protocols/irc/dcc-send.zeek

```diff
@@ -44,26 +44,29 @@ function dcc_relay_topic(): string &is_used
 	return rval;
 	}
 
+const have_cluster = Cluster::is_enabled();
+const should_publish =
```

Comment:
I stumbled a bit over this pattern that's used elsewhere, no need to address because it's again in the naming and subjective area, but wanted to leave the thoughts.

* Are you worried that a `Cluster::is_enabled()` call is too expensive, and/or that caching seems a reasonable optimization?

* "have cluster" and "Cluster::is_enabled()", is close, but it's still different enough for me to go back the first few times to ensure it's just Cluster::is_enabled(). Is `const cluster_is_enabled =` too long? We could probably also cache a `Cluster::is_enabled_value` const?

* About "should_publish". Workers do call Broker::publish() just as well and seems if it's a global all Broker::publish() should be protected with it. Proxies / managers responsibility is to forward/broadcast/relay messages to workers, which I think this is representing? should_forward / should_relay?

Reply:
Happy to go with `cluster_is_enabled` (and now added to my pending update). Even though in this case it's trivial, I think of the base scripts as modeling patterns that script writers will often follow, and the pattern of "if you use the conditional more than once, put it in a `const` so it's immediately apparent that the two uses are the same" is a good one.

Regarding `should_publish`, @awelzel I take your comment to be more about is-the-current(pre-PR)-structure-in-fact-right. As the PR isn't about cluster structural logic but just about introducing the new feature where it's apt, I'm thinking I won't bite off the broader notion of just what `should_publish` should cover.

Reply:
> pattern of "if you use the conditional more than once, put it in a const so it's immediately apparent that the two uses are the same" is a good one.

 :+1: 

> @awelzel I take your comment to be more about is-the-current(pre-PR)-structure-in-fact-right.

Nah, I meant that the "should_publish" const name is a too generic for it's actual purpose. It's scoped small enough, not a module API and a naming topic, so I'm resolving this discussion.